pkgs/applications/version-management/git-and-tools/git-machete/default.nix

```diff
@@ -22,19 +22,37 @@ buildPythonApplication rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  checkInputs = [ git stestr ];
+  checkInputs = [ git pytest ];
 
+  ### Maybe not needed at all? since `python setup.py test` is executed anyway?
+  /*
+  Executing setuptoolsCheckPhase
+  running test
+  WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
+  running egg_info
+  writing git_machete.egg-info/PKG-INFO
+  writing dependency_links to git_machete.egg-info/dependency_links.txt
+  writing top-level names to git_machete.egg-info/top_level.txt
+  reading manifest file 'git_machete.egg-info/SOURCES.txt'
+  reading manifest template 'MANIFEST.in'
+  adding license file 'LICENSE'
+  writing manifest file 'git_machete.egg-info/SOURCES.txt'
+  running build_ext
+  test_add (git_machete.tests.functional.test_machete.MacheteTester)
+  Verify behaviour of a 'git machete add' command. ... ok
+  ....
+  */
   postCheck = ''
-    stestr run
+    pytest
   '';
 
   postInstall = ''
     installShellCompletion --bash --name git-machete completion/git-machete.completion.bash
     installShellCompletion --zsh --name _git-machete completion/git-machete.completion.zsh
+    installShellCompletion --fish completion/git-machete.fish
   '';
 
   postInstallCheck = ''
-    git init
```

Comment:
No longer needed after the fix on https://github.com/VirtusLab/git-machete/pull/381